### ISSUE TITLE
Train Model 5: Unknown Root Classifier (500K params, OPTIONAL)

### ISSUE THREAD

**Issue by marctjones** [marctjones/klareco#669]
## Goal

**OPTIONAL**: Only train if Phase 2 annotations (500 roots) achieve <90% corpus coverage.

Predict semantic classes for unknown roots (tier3, not in annotations) using character similarity, context, and morphology.

## Architecture

```python
class UnknownRootClassifier(nn.Module):
    # 500K parameters
    # Character-level LSTM (morphological similarity)
    # Context encoder (other roots in sentence)
    # Affix encoder (compositional features)
    # Classifier (150 semantic classes)
```

## When It Helps

```
Unknown root: "instauris" (established - rare form)
Not in our 500-1000 annotated roots

Character similarity: similar to "instauri", "establi", "fondi" → creation verbs
Context: "regnestro instauris novan leĝon" → ruler + law = creation context
Morphology: no special affixes

Predicted: "kreado-26" (creation) with 0.75 confidence
```

## Training Data Needed

2,000-5,000 examples: (unknown_root, context, gold_semantic_class)

**Collection strategy**:
1. Hold out 20% of annotated roots as "unknown"
2. Train on 80%, test on held-out 20%
3. Ensures model learns to generalize

## Training Approach

```python
# Cross-entropy loss
loss = nn.NLLLoss()(predicted_class_logits, gold_class_id)
```

## Quality Metrics

- Top-1 accuracy: >70% (correct class in top prediction)
- Top-3 accuracy: >85% (correct class in top-3)
- Coverage improvement: +5-10% corpus coverage

## Decision Point

After Phase 2 (500 roots annotated):
- ✅ Coverage <90% → Train this model
- ❌ Coverage ≥90% → Skip this model (not needed!)

## Dependencies

- #659 (500 roots annotation - must measure coverage first)
- Root Embeddings (#479)
- #655 (Semantic schema with all classes defined)

## Timeline

2 weeks (Phase 2, Week 7-8) - **only if needed**

## Related

- See: docs/COMPLETE_SYSTEM_DESIGN_WITH_MODELS.md (Model 5)
- Part of Epic #654

**Comment by marctjones** [comment]
Closing as part of a 2026-09-08 backlog cleanup (see PLAN.md, DESIGN.md, CLAUDE.md's merge gate).

This issue predates the merge gate (adopted 2026-07-13) and the AST-native orchestrator rewrite. It describes work against the pre-orchestrator RAG pipeline or the learned model stack (M1/M2 embeddings, compositional embeddings, entity classifiers, neural rerankers) that was pruned from the repo -- klareco/embeddings/, klareco/models/, and klareco/summarization/ no longer exist. It has no milestone and was never benchmarked, matching exactly the failure mode CLAUDE.md documents (25 capability issues filed 2026-05-26, 8 implemented, none benchmarked).

Labeled archived-2026-09 so this batch is recoverable: gh issue list --state closed --label archived-2026-09. If this capability is still wanted, re-file it against the current architecture naming the benchmark metric and test the merge gate requires.
